project/src/main/ui/settings/SettingsMenu.tscn

```diff
@@ -858,7 +858,7 @@ size_flags_horizontal = 3
 size_flags_vertical = 4
 max_value = 11.0
```

Comment:
max_value shouldn't be 11.0; it should either be the correct value (10) or it should be unset

Reply:
Removed; max_value and tick_count have their defaults in the Godot Editor, and are only assigned when running the game.